policy/modules/system/logging.if

```diff
@@ -367,6 +367,25 @@ interface(`logging_manage_audit_log',`
 	dontaudit $1 auditd_log_t:file map;
 ')
 
+########################################
+## <summary>
+##	Relabel from and to audit log directory type.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+## <rolecap/>
+#
+interface(`logging_relabel_audit_log_dirs',`
+	gen_require(`
+		type auditd_log_t;
+	')
+
+	allow $1 auditd_log_t:dir relabel_dir_perms;
+')
+
```

Comment:
Please drop this too.

Reply:
Good catch, I forgot about that one... Thanks!